### ISSUE TITLE
[FEAT.] Hyperion

### ISSUE THREAD

**Issue by Synthbit** [Fadi002/de4py#1]
**Is your feature request related to a problem? Please describe.**
No, just idea

**Describe the solution you'd like**
Add support for hyperion, (https://github.com/bilythegoat356/Hyperion)
I already made deobfuscator if yall want i can provide src, but python is my favorite language. Adding Hyperion would be fine

**Additional context**
Not any, cuz everyone who coded before in python should know Hyperion..

also if Hyperion support is in there, im clearly sorry i havent seen it.. mostly blank obfuscator for blank grabber, jawbreaker etc..

**Comment by Fadi002** [comment]
Sure, I have exams rn when i finish I will add support for them.
Also if you can give me the deobfuscator it will be very helpful for me

**Comment by Synthbit** [comment]
Sure, np i did factory reset while ago. i might need to rewrite but before that ill check github if theres any so i dont waste my time.

https://github.com/xKiian/Hyperion-deobfuscator
https://github.com/teaishealthy/erebus

check these out.. or your partners that were participants in the repository :) if it doesnt work ill make own  👍

**Comment by Fadi002** [comment]
> Sure, np i did factory reset while ago. i might need to rewrite but before that ill check github if theres any so i dont waste my time.
> 
> Edit, I found some:
> 
> https://github.com/xKiian/Hyperion-deobfuscator https://github.com/teaishealthy/erebus
> 
> check these out.. or your partners that were participants in the repository :) if it doesnt work ill make own 👍

btw this is not even the real repo lol
https://github.com/billythegoat356/Hyperion
this is the real repo

**Comment by Synthbit** [comment]
yes my bad, the guy who i sented is some kind of larp..

**Comment by AdvDebug** [comment]
> > Sure, np i did factory reset while ago. i might need to rewrite but before that ill check github if theres any so i dont waste my time.
> > Edit, I found some:
> > https://github.com/xKiian/Hyperion-deobfuscator https://github.com/teaishealthy/erebus
> > check these out.. or your partners that were participants in the repository :) if it doesnt work ill make own 👍
> 
> btw this is not even the real repo lol https://github.com/billythegoat356/Hyperion this is the real repo

it's good that you noticed, a lot of people started making fake issues to try to trick other github users to install malware/stealers.

**Comment by Synthbit** [comment]
sorry it was my bad, it was odd that billy had 4 stars or how much it was.. cuz hyperion was popular and now i see that its bily and not billy once again my bad guys

**Comment by Synthbit** [comment]
anyways i like nomorecookies repo i might try to make remover for it :D maybe very hard havent checked it fully

**Comment by AdvDebug** [comment]
> anyways i like nomorecookies repo i might try to make remover for it :D maybe very hard havent checked it fully

good luck with that xD

**Comment by AdvDebug** [comment]
you will need more than chatgpt and google to "remove" it

**Comment by Synthbit** [comment]
chatgpt is bad, didnt said before it will be easy tried it before.. was hard but ill try once again, atleast the mini ver

**Comment by AdvDebug** [comment]
> chatgpt is bad, didnt said before it will be easy tried it before.. was hard but ill try once again, atleast the mini ver

yeah you can bypass the mini version since it have no anti-tamper, just a simple protection. anyway, best of luck bro.

**Comment by Synthbit** [comment]
Lets see if i can do it :D, ty for the luck tho..

**Comment by Synthbit** [comment]
Done second phase, like hr ago, the third phase is rly but rly hard

**Comment by AdvDebug** [comment]
> Done second phase, like hr ago, the third phase is rly but rly hard 1,2 = Done 3 = Hard i dont have time for it 💀

yup for the two you just delete AppInit_DLLs because i forgot to prevent deletion of AppInit_DLLs in the second protection before archiving it, but even if you had time you wouldn't bypass it.

**Comment by Synthbit** [comment]
dang, but theres gotta be way somehow.. 💀  if i do im best!!

**Comment by AdvDebug** [comment]
i'm hardening it right now anyway, i will fix that bug

**Comment by Synthbit** [comment]
NOOOO!!! I THOUGHT YOU WONT UPD ANYMORE

**Comment by AdvDebug** [comment]
well, that bug is too stupid to leave it like that lol

**Comment by AdvDebug** [comment]
thanks for getting this to my attention tho

**Comment by Synthbit** [comment]
yeah no problem..  :)
